app/assets/stylesheets/modules/show_playdates.scss

```diff
@@ -0,0 +1,27 @@
+html, body {
+  height: 100%;
+  margin: 0;
+}
+
+#map {
+  width: 450px;
+  height: 550px;
+  border-width: 7px;
+  box-shadow: 2px 2px 10px #B1B1B1;
+  border-style: solid;
+
+}
+
+
+
+#map-container {
+   height: 400px;
```

Comment:
Line should be indented 2 spaces, but was indented 3 spaces<br>Properties should be ordered border-color, border-radius, height, margin-top, width